test/runtests.jl

```diff
@@ -32,21 +33,25 @@ end
     @test Showoff.format_fixed_scientific(Inf, 1, false) == "∞"
     @test Showoff.format_fixed_scientific(-Inf, 1, false) == "-∞"
     @test Showoff.format_fixed_scientific(NaN, 1, false) == "NaN"
-    @test Showoff.format_fixed_scientific(0.012345678, 4, true) == "12.34568×10⁻³"
-    @test Showoff.format_fixed_scientific(0.012345678, 4, false) == "1.234568×10⁻²"
-    @test Showoff.format_fixed_scientific(-10.0, 4, false) == "-1.000×10¹"
+    @test Showoff.format_fixed_scientific(0.012345678, 4, true) == "12.346×10⁻³"
```

Comment:
Very nice, the original seems borked. I was worried about this originally (4 digits after the decimal?), but with engineering notation this seems to be as good a choice as any. It might be worth adding a brief comment in the code that the precision applies *pre*-engineering-ification (or perhaps more simply that `precision+1` is the number of mantissa digits printed).

Reply:
Right, I will add the comment